gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/ImageService.groovy

```diff
@@ -37,7 +37,7 @@ class ImageService {
     Retrofit2SyncCall.execute(clouddriverServiceSelector.select().getImageDetails(provider, account, region, imageId))
   }
 
-  List<Map> search(String provider, String query, String region, String account, Integer count, Map<String, Object> additionalFilters, String selectorKey) {
+  List<Map> search(String provider, String query, String region, String account, Integer count, Map<String, String> additionalFilters, String selectorKey) {
```

Comment:
Apparently the only call to this is from [ImageController.findImages](https://github.com/spinnaker/gate/blob/381f554c99db24dca82e254bf2fd7323506b268a/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ImageController.groovy#L55-L57) and the value in the Map it uses is a String, so this seems safe.

Reply:
Well, I merged this too soon...I don't think it's sufficient.  We're missing more test coverage and groovy isn't helping.  That code in ImageController is gonna fail with
```
'search' in 'com. netflix. spinnaker. gate. services. ImageService' cannot be applied to '(java. lang. String, java. lang. String, java. lang. String, java. lang. String, java. lang. Integer, java. util. Map<java. lang. Object,java. lang. Object>, java. lang. String)' 
```
I'm looking in to a test/fix.

Reply:
Apparently https://github.com/spinnaker/gate/pull/1877 isn't required, but it makes me feel better.